disk/development/ontology.ttl

```diff
@@ -302,6 +326,53 @@ xsd:date rdf:type rdfs:Datatype .
                     rdfs:comment "Property that links a line of inquiry to the hypothesis query text"@en ;
                     rdfs:label "has hypothesis query"@en .
 
+###  http://disk-project.org/ontology/disk#hasEmail
+:hasEmail rdf:type owl:DatatypeProperty ;
+                 rdfs:domain :Person ;
+                 rdfs:range xsd:string ;
+                 rdfs:comment "Email of a person"@en ;
+                 rdfs:label "has email"@en .
+
+###  http://disk-project.org/ontology/disk#hasName
+:hasName rdf:type owl:DatatypeProperty ;
+                 rdfs:domain :Person ;
+                 rdfs:range xsd:string ;
+                 rdfs:comment "Name of a person"@en ;
+                 rdfs:label "has name"@en .
+
+###  http://disk-project.org/ontology/disk#hasUsageNotes
+:hasUsageNotes rdf:type owl:DatatypeProperty ;
+             rdfs:domain [ rdf:type owl:Class ;
+                                      owl:unionOf ( :LineOfInquiry
+                                                    :Hypothesis
+                                                  )
+                                    ] ;
+             rdfs:range xsd:string ;
+             rdfs:comment "Property that add notes to Line of Inquiry and Hypothesis"@en ;
```

Comment:
It doesn't add notes. It's a property to indicate what are the usage notes to take into consideration when using a line of inquiry or a hypothesis